docs/installation/cloud/overview.md

```diff
@@ -26,11 +26,11 @@ To install on a cloud provider:
 
 2. Decide which OS you want to run on the cloud host.
```

Comment:
I wanted to give a little more info in the title of the page than in the menu where we have less space, but I will change this per your preference.